bazel/toolchains/configs/clang_libcxx/bazel_0.28.1/cc/BUILD

```diff
@@ -0,0 +1,149 @@
+# Copyright 2016 The Bazel Authors. All rights reserved.
```

Comment:
Do you think you can move this to some other repository that we consume, to reduce the churn in the main Envoy repo? Just a suggestion, we can ship this PR as is.

Reply:
Yes it is totally possible to have whole `bazel/toolchains/configs` in a separate repo, or even just a generated tar in somewhere (like the build image).  It just need more bash snafu to automate the generate process though.

I don't expect much more churn from here other than every bazel update (likely it will be just rename based on Bazel version) or adding new configuration (e.g. MSAN)